crates/blockchain/blockchain.rs

```diff
@@ -28,10 +28,18 @@ use ethereum_rust_vm::{
 ///
 /// Performs pre and post execution validation, and updates the database with the post state.
 pub fn add_block(block: &Block, storage: &Store) -> Result<(), ChainError> {
-    // TODO(#438): handle cases where blocks are missing between the canonical chain and the block.
+    let block_hash = block.header.compute_block_hash();
```

Comment:
nit, should we make this a function for block? Maybe in the future we will be able to cache it. block.block_hash() or something

Reply:
Sure, I can do that. I'll add a quick separate PR for that if that's ok!